app/controllers/packages_controller.rb

```diff
@@ -11,14 +15,31 @@ def show
     render jsonapi: @package, include: params[:include]
   end
 
+  def update
+    @package = packages.find package_id
+    @package.update package_params
```

Comment:
The "AR" way to do this would be:

```ruby
@package = packages.update package_id, package_params
render status: :no_content
```
I know that's not the way that customer resources does it, but worth considering changing them both.... or maybe with the next update.

Reply:
Let's revisit that later.